src/ts/components/buttons/Button.ts

```diff
@@ -77,6 +77,10 @@ export class Button<Config extends ButtonConfig> extends Component<Config> {
       this.onClickEvent();
     });
 
+    buttonElement.on('focusin focusout', (e) => {
+      e.stopPropagation();
+    });
```

Comment:
This is needed to stop bubbling of the `focusin` event to parents. This is a problem in the new settings panel where the subtitle settings button is inside a focusable settings panel item. Without that the `focusin` fires on both elements which is undesired.